docs/02-get-started-with-mini.md

```diff
@@ -0,0 +1,111 @@
+# Get started with Mini
+
+Serverpod Mini is a slimmer version of Serverpod that does not need to be connected to a Postgres database. Before you begin, make sure that you have __[Flutter](https://flutter.dev/docs/get-started/install)__ and __[Serverpod](/)__ installed.
+
+Create a mini project by running:
+
+```bash
+$ serverpod create myminipod --mini
+```
+
+Serverpod will create a new project for you. It contains three Dart packages, but you only need to pay attention to the `myminipod_server` and `myminipod_flutter` directories. The server directory contains your server files, and the flutter directory contains your app. The third package (`myminipod_client`) contains generated code that is used by the Flutter app to communicate with the server.
+
+Start your server by changing directory into your server directory, and run the `bin/main.dart` file:
+
+```bash
+$ cd myminipod/myminipod_server
+$ dart bin/main.dart
+```
+
+Your default project comes with a sample Flutter app, all hooked up to talk with your server. Run it with the `flutter` command:
+
+```bash
+$ cd myminipod/myminipod_flutter
+$ flutter run -d chrome
+```
+
+Easy as that. 🥳
+
+:::tip
+
+If you are using VS Code, install our Serverpod extension. It will help you validate any Serverpod-related files in your project!
+
+:::
+
+## Creating models
+In Serverpod, you define your models in easy-to-read YAML-files, which you place in your server’s `lib/src/models` directory. Model files will be converted to Dart classes that can be serialized and sent to and from the server to your app. This is an example of a model file:
+
+```yaml
+class: Company
+fields:
+  name: String
+  foundedDate: DateTime?
+  employees: List<String>
+```
+
+For types, you can use most basic Dart types, such as `String`, `double`, `int`, `bool`, `DateTime`, and `ByteData`. You can also include `List` and `Map`, just make sure to specify their types. Any other class specified among your models is also supported.
+
+Whenever you add or edit a model file, run `serverpod generate` in your server directory. Then, Serverpod will generate all the updated Dart classes:
+
+```bash
+$ cd myminipod/myminipod_server
+$ serverpod generate
+```
+
+## Adding methods to your server
```

Comment:
Comment: The story based instructions are great! We should make this improvement to the non mini "Get Started" page as well.